### PR TITLE
Camera_sensor

### PR DESCRIPTION
**Pull request by mohamedsayed18**

### CODE REVIEW

File: dogspace/src/spot/scripts/camera_sub.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  def __init__(self):
+    #self.image_pub = rospy.Publisher("image_topic_2",Image)
+    self.counter = 0
+    self.bridge = CvBridge()
+    self.image_sub = rospy.Subscriber("image_raw",Image,self.callback)
+
+  def callback(self,data):
+    try:
+      cv_image = self.bridge.imgmsg_to_cv2(data, "bgr8")
+      cv2.imwrite('/media/mohamed/C03CCDB43CCDA62E1/tutorials/Innopolis/1st/FRP/HA/Robot_Dog/images/image'+str(self.counter)+'.png', cv_image)
```
Comment (l1va) [inline dogspace/src/spot/scripts/camera_sub.py]: hardcode that will not work on my laptop

Comment (mohamedsayed18) [reply dogspace/src/spot/scripts/camera_sub.py]: so, one of the best practices is not to use a specific directory for me, just a general one ?

Comment (l1va) [reply dogspace/src/spot/scripts/camera_sub.py]: use relative paths, not absolute

Comment (mohamedsayed18) [reply dogspace/src/spot/scripts/camera_sub.py]: edited

Comment (l1va) [reply dogspace/src/spot/scripts/camera_sub.py]: at least like this, yes. The best way like: getCurrentPath() inside the script and add /images.

**Pull request merged**